app/v1/connectors/AmendPensionsReliefsConnector.scala

```diff
@@ -29,12 +30,21 @@ import scala.concurrent.{ExecutionContext, Future}
 @Singleton
 class AmendPensionsReliefsConnector @Inject() (val http: HttpClient, val appConfig: AppConfig) extends BaseDownstreamConnector {
 
-  def amend(
-      request: AmendPensionsReliefsRequest)(implicit hc: HeaderCarrier, ec: ExecutionContext, correlationId: String): Future[DownstreamOutcome[Unit]] = {
+  def createOrAmendPensionsRelief(request: AmendPensionsReliefsRequest)(implicit
+      hc: HeaderCarrier,
+      ec: ExecutionContext,
+      correlationId: String): Future[DownstreamOutcome[Unit]] = {
+
+    val downstreamUri =
+      if (request.taxYear.useTaxYearSpecificApi) {
+        TaxYearSpecificIfsUri[Unit](s"income-tax/reliefs/pensions/${request.taxYear.asTysDownstream}/${request.nino.nino}")
+      } else {
+        DesUri[Unit](s"income-tax/reliefs/pensions/${request.nino.nino}/${request.taxYear.asMtd}")
```

Comment:
The format of taxYear '.asTysDownstream' and '.asMtd' is correct here, but the comments for '.asMtd' (in TaxYear.scala) has the wrong format. I did change this myself but seems to still be wrong. Could you change these comments again please.

Reply:
did a rebase